webapp/pages/index.vue

```diff
@@ -190,6 +190,11 @@ export default {
 </script>
 
 <style lang="scss">
+.ds-card-image img {
+  max-height: 2000px;
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-10-14T16:50:23Z" title="Monday, October 14th 2019, 6:50:23 pm +02:00">Oct 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -190,6 +190,10 @@ export default {
 </script>
 
 <style lang="scss">
+.ds-card-image img {
+  max-height: 2000px;
```


```suggestion
  max-height: 2000px;
  object-fit: contain;
```

aah, we're defining this `.ds-card-image img` in so many places – I'm not even sure where the `object-fit` needs to go to get applied in the right places! but I guess we'll need it everywhere.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-14T17:13:45Z" title="Monday, October 14th 2019, 7:13:45 pm +02:00">Oct 14, 2019</time>_

---


yeah, there are some that are scoped, for other CSS reasons I guess :worried: